include/RE/M/MenuTopicManager.cpp

```diff
@@ -0,0 +1,11 @@
+#include "RE/M/MenuTopicManager.h"
+#include "RE/N/NiSmartPointer.h"
+
+namespace RE
+{
+	MenuTopicManager* MenuTopicManager::GetSingleton()
+	{
+		REL::Relocation<NiPointer<MenuTopicManager>*> singleton{ REL::ID(514959) };
+		return singleton->get();
+	}
```

Comment:
Can you inline this in the header? I'd like to avoid compiling additional source files where possible.

Reply:
Is the singleton instance wrapped in a `NiPointer` though? The NetImmerse smart pointer class is only used with intrusively ref counted classes, IIRC. `MenuTopicManager` isn't one of those, as far as I can tell.

I just have it as a bare pointer in my code.